platform-sdk/swirlds-merkle/src/timingSensitive/java/com/swirlds/virtual/merkle/map/MapTest.java

```diff
@@ -119,11 +118,8 @@ void insertRemoveAndModifyOneMillion() throws InterruptedException {
             // Make sure all the created virtual map copies are fully processed before this test
             // is complete, otherwise OOME can be observed in random tests run after this one
             final VirtualRootNode<TestKey, TestValue> root = map.getRight();
-            root.enableFlush();
-            final VirtualMap<TestKey, TestValue> lastCopy = map.copy();
             map.release();
-            lastCopy.release();
-            assertEventuallyTrue(root::isFlushed, Duration.ofMinutes(1), "The map must be flushed");
+            assertTrue(root.getPipeline().awaitTermination(30, SECONDS), "Pipeline termination timed out");
```

Comment:
Let's keep it 1 minute. It used to be 30 seconds before, and it was not enough in some cases, see https://github.com/hiero-ledger/hiero-consensus-node/pull/18407 for details